Description: Include debian as extra directory instead of directory part of
 subdirectories used in build.
Origin: http://anonscm.debian.org/gitweb/?p=pkg-multimedia/lame.git;a=tree;f=debian/patches
Forwarded: yes
Applied-Upstream: http://lame.cvs.sf.net/viewvc/lame/lame/Makefile.am?revision=1.39

--- a/Makefile.am
+++ b/Makefile.am
@@ -4,7 +4,7 @@
 
 .PHONY: test
 
-SUBDIRS = mpglib libmp3lame frontend Dll debian doc include misc dshow ACM \
+SUBDIRS = mpglib libmp3lame frontend Dll doc include misc dshow ACM \
 	mac macosx vc_solution
 
 CLEANFILES = testcase.new.mp3
@@ -22,6 +22,7 @@
 	STYLEGUIDE \
 	USAGE \
 	configMS.h \
+	debian \
 	lame.bat \
 	lame.spec.in \
 	lame.spec \
--- a/configure.in
+++ b/configure.in
@@ -1152,6 +1152,6 @@
 
 AC_SUBST(CONFIG_DEFS)
 
-AC_CONFIG_FILES([Makefile libmp3lame/Makefile libmp3lame/i386/Makefile libmp3lame/vector/Makefile frontend/Makefile mpglib/Makefile doc/Makefile doc/html/Makefile doc/man/Makefile include/Makefile Dll/Makefile misc/Makefile debian/Makefile dshow/Makefile ACM/Makefile ACM/ADbg/Makefile ACM/ddk/Makefile ACM/tinyxml/Makefile lame.spec mac/Makefile macosx/Makefile macosx/English.lproj/Makefile macosx/LAME.xcodeproj/Makefile vc_solution/Makefile])
+AC_CONFIG_FILES([Makefile libmp3lame/Makefile libmp3lame/i386/Makefile libmp3lame/vector/Makefile frontend/Makefile mpglib/Makefile doc/Makefile doc/html/Makefile doc/man/Makefile include/Makefile Dll/Makefile misc/Makefile dshow/Makefile ACM/Makefile ACM/ADbg/Makefile ACM/ddk/Makefile ACM/tinyxml/Makefile lame.spec mac/Makefile macosx/Makefile macosx/English.lproj/Makefile macosx/LAME.xcodeproj/Makefile vc_solution/Makefile])
 
 AC_OUTPUT
